Examples/Detectors/ContextualDetector/include/ActsExamples/ContextualDetector/ExternalAlignmentDecorator.hpp

```diff
@@ -65,10 +61,10 @@ class ExternalAlignmentDecorator : public AlignmentDecorator {
   /// added in order.
   ///
   /// @param context the bare (or at least non-const) Event context
-  ProcessCode decorate(AlgorithmContext& context) override;
+  ProcessCode decorate(AlgorithmContext& context) final override;
```

Comment:
Generally we only do one out of `virtual`, `override` and `final`, as `final` implies `override`.